### ISSUE TITLE
No-Reference Quality Assessment.

### ISSUE THREAD

**Issue by Body123** [Retinal-Research/OTE-GAN#3]
Question 1:

I'm trying to understand the CR metric. Does it represent the percentage of all enhanced images that are considered high-quality (compared to both high-quality and usable results)?

Question 2:

The other metrics mentioned (Accuracy, Kappa, AUC) seem to relate to a model trained for classifying images as high-quality, usable, or rejected only and there is no relation in enhancement here am I missed something?

![15](https://github.com/Retinal-Research/OTE-GAN/assets/24945076/7334c40f-43db-4773-8533-e0c6e5b93b93)

**Comment by ChongQingNoSubway** [comment]
Thank you very much for your interest in our work. 

(1) Here, CR refers to conversion rate as a separate task. Essentially, we enhance rejected retinal images, then utilize a pre-trained quality classification model (you could directly use the quality label classification model in EyeQ: https://github.com/HzFu/EyeQ/tree/master/MCF_Net) for direct inference to determine how many images are converted to high quality.

(2).  Accuracy, Kappa, and AUC are metrics for another classification task.  we use enhanced images for the DR grading task (DR grading label in eye/eyepieces CSV) to check the lesion-preserving quality. Here, we employ a straightforward ResNet-50 model.

**Comment by Body123** [comment]
Point 2 presents two options for classifying enhanced images in the context of diabetic retinopathy (DR):

Direct inference: A pre-trained model, originally trained on normal images, can be directly applied to predict DR on enhanced images.
Model retraining: Alternatively, the enhanced images can be used to train a new model specifically for DR classification.

so the results in the paper related to what of them ?

**Comment by ChongQingNoSubway** [comment]
Model retraining

**Comment by Body123** [comment]
was that based on something?

**Comment by ChongQingNoSubway** [comment]
It's all about training from scratch here; you just have to fix the standard backbone and training configuration for all enhanced models. You can use our latest nn-mobilenet, where eyepacs dataloader and eyeq are the same. (https://github.com/Retinal-Research/NN-MOBILENET)
